backend/src/main/java/com/bakdata/conquery/sql/conversion/cqelement/concept/EventFilterCte.java

```diff
@@ -16,17 +17,28 @@
 import com.bakdata.conquery.sql.conversion.model.select.FieldWrapper;
 import com.bakdata.conquery.sql.conversion.model.select.SqlSelect;
 import com.google.common.base.Preconditions;
+import org.jetbrains.annotations.NotNull;
 import org.jooq.Condition;
+import org.jooq.Field;
 
 class EventFilterCte extends ConnectorCte {
 
 	@Override
 	public QueryStep.QueryStepBuilder convertStep(CQTableContext tableContext) {
+
+		List<Condition> conditions = new ArrayList<>();
+
+		//TODO refactor CQTableContext for PreprocessingCte: Move this filter to PPCte. This would mean passing down the raw ids.
```

Comment:
Hier auch das gleiche Problem. Die Abstraktion versteckt die Ids komplett in der PreprocessingCte. Das braucht ein Refactoring was zu groß ist